src/components/InteriorLeftNav/InteriorLeftNav-story.js

```diff
@@ -4,6 +4,9 @@ import InteriorLeftNav from '../InteriorLeftNav';
 import InteriorLeftNavItem from '../InteriorLeftNavItem';
 import InteriorLeftNavList from '../InteriorLeftNavList';
 
+// For demo purposes
+import { BrowserRouter, Link } from 'react-router-dom';
```

Comment:
Should we add these into a test too where we verify `Link` works inside of ILNI?

Reply:
If we go with the solution proposed above, I'll remove this :)